src/profiler.cpp

```diff
@@ -1819,7 +1825,7 @@ void Profiler::stopTimer() {
 
 void Profiler::timerLoop(void* timer_id) {
     u64 current_micros = OS::micros();
-    u64 stop_micros = _stop_time * 1000000ULL;
+    u64 stop_micros = _stop_time;
```

Comment:
I guess we don't need this local variable anymore now, let's just use `_stop_time`